crypto/x509v3/v3_conf.c

```diff
@@ -313,8 +313,12 @@ int X509V3_EXT_add_nconf_sk(CONF *conf, X509V3_CTX *ctx, const char *section,
             return 0;
         if (ctx->flags == X509V3_CTX_REPLACE)
             delete_ext(*sk, ext);
-        if (sk)
-            X509v3_add_ext(sk, ext, -1);
+        if (sk) {
```

Comment:
Could we make this `sk != NULL`. That is our preferred style where the variable being tested is not a boolean. We fix these as we come across them.

Reply:
I checked it only on master.

Reply:
I also added one more patch which I initially forgot to include into the pull request.